feat(guidelines): add new guideline for money fields (#89)
otto-de/api-guidelines: api-guidelines/global/json/canonical-data-types/rules/should-use-common-money-object.md

> ---

Reviewer: Can you please add the "otto" part to the file name so that the file name matches the title? :)

id: R100080
---

# SHOULD use common money object

> As money amounts are very common in our APIs (checkout, order-positions, transactional-communication), this rule ensures that money amounts are always formatted the same.

Reviewer 2: @cgebken I merged both of our suggestions in my comment below. You can delete this one.

> As money amounts are commonly used in our APIs, for example, in the `Checkout`, Order management`, and `Transactional communication` API, this guideline ensures that money amounts are always formatted in the same way.

Reviewer: As it's a SHOULD and not a MUST rule, it doesn't really "ensure"...

```suggestion
As money amounts are commonly used in our APIs, for example, in the `Checkout`, `Order management`, and `Transactional communication` API, this guideline aims to ensure that money amounts are always formatted in the same way.
```
